docs/getting-started/f44rm-how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+slug: /f44rm/how-it-works
+displayed_sidebar: "0.2"
+id: how-it-works
+---
+
+import Button from "../../website/src/components/atoms/button.js";
+import CardGrid from "../../website/src/components/molecules/cardGrid.js";
```

Comment:
Same comment as the templates one. The component is only used here. No need to create a component yet.